### DISCUSSION TITLE
Copy to Clone of List with Different Name and Path

### DISCUSSION THREAD

**Discussion by mushieslushie** [Zerg00s/FlowPowerAppsMigrator#35] [Ideas]
Hello, I came across this tool and while it seems nice, I don't thinks it helps me with my particular issue.

I'm looking to copy a list and form, just with a different name.  This tool only seems to copy to another site if the names and structure in the site is exactly the same.

I have found that in that scenario, exporting, deleting the data connections and adding them back (in Power Apps) is relatively quick and easy even without this tool (can't say the same for Power Automate).

Unfortunately I think I am go to have to rebuild my App mostly from scratch to be able to associate it to a list with a different name.

**Comment by Zerg00s** [comment]
Hi @mushieslushie . I hear you. Yes, so far it's not easy to do what you are saying using this script. However, with some hacking you can achieve it:
- Migrate or re-create your SharePoint list with a different name of your choice
- Run the GenerateInitialMapping.ps1 script
- Run the CompleteResourceMapping.ps1 script
- Manually open and update the resourceMapping.csv file. Make sure that the old GUIDs for the original list map to the new GUIDS for your new, renamed list
- Run the ConvertPackage.ps1 script to convert your power app package based on the resourceMapping.csv file.

You are not the first person who mentions this feature. Perhaps I should find some time to develop it. Although I'm not yet sure when I will be able to do it

**Reply by mushieslushie** [reply]
OK thanks, I did see the csv and assumed you could do something with it, but I didn't see instructions for how to run again it using a modified csv.  Thanks for the quick response.
